ibm_cloud_sdk_core/base_service.py

```diff
@@ -98,6 +105,22 @@ def __init__(self,
             raise ValueError(
                 'authenticator should be of type Authenticator')
 
+    def setup_http_client(self) -> None:
```

Comment:
This would effectively become the `enable_retries(max_retries, retry_interval)` method.